query-connector/src/app/backend/usergroup-management.ts

```diff
@@ -189,26 +187,21 @@ export async function addUsersToGroup(
       `,
       values: [membershipIds, userIds, groupIds],
     };
-    console.log("kcd 3", insertQuery);
 
     const result = await dbClient.query(insertQuery);
-    console.log("result.rows", result.rows);
 
     const updatedUsers = await Promise.all(
       result.rows.map(async (updatedUser) => {
-        console.log("1 updatedUser", updatedUser);
-
         const updatedUserWithGroups = await getSingleUserWithGroupMemberships(
           updatedUser.user_id,
         );
-        console.log("updatedUserWithGroups", updatedUserWithGroups);
+        await dbClient.query("COMMIT");
```

Comment:
assuming we're adding this because  interior db write was causing a race condition // what was causing the test flakage? if so, nice catch! 
 
down the line as we rework this into a service class, wondering if we could take advantage of the transaction decorator to prevent things like this in the future